src/oauth.test.ts

```diff
@@ -19,15 +21,26 @@ import {
 } from "../tests/helpers/oauth";
 import { getLoginCookie } from "../tests/helpers/web";
 import { OOB_REDIRECT_URI } from "./oauth/constants";
-import { createAccessGrant } from "./oauth/helpers";
+import {
+  calculatePKCECodeChallenge,
+  createAccessGrant,
+  generatePKCECodeVerifier,
+} from "./oauth/helpers";
+
+async function getPage(response: Response) {
+  const text = await response.text();
+  const { document } = parseHTML(text);
+
+  return document;
+}
```

Comment:
Eventually we can move this to `test/helpers` or something, once we expand view testing.